src/user/dumbbell_initialization.F90

```diff
@@ -231,12 +265,18 @@ subroutine dumbbell_initialize_temperature_salinity ( T, S, h, G, GV, US, param_
   real    :: x         ! The fractional position in the domain [nondim]
   real    :: dblen     ! The size of the dumbbell test case [axis_units]
   logical :: dbrotate  ! If true, rotate the domain.
+  logical :: use_ALE     ! If false, use layer mode.
   character(len=20) :: verticalCoordinate, density_profile
 
   is = G%isc ; ie = G%iec ; js = G%jsc ; je = G%jec ; nz = GV%ke
 
   T_surf = 20.0*US%degC_to_C
 
+  ! layer mode
+  call get_param(param_file, mdl, "USE_REGRIDDING", use_ALE, do_not_log = .true.)
+  if(.not. use_ALE) call MOM_error(FATAL,  "dumbbell_initialize_temperature_salinity: "//&
```

Comment:
As with line 136, please add a space between if and the following parenthesis, as discussed in the MOM6 style guide at https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide#white-space.